planetiler-custommap/src/test/java/com/onthegomap/planetiler/custommap/ConfiguredFeatureTest.java

```diff
@@ -1022,4 +1112,78 @@ void testUseArgumentInSourceUrlPath() {
     )));
     assertEquals("example.com_file.osm.pbf", loadConfig(config).sources().get(0).defaultFileUrl());
   }
+
+  @Test
+  void testSchemaEmptyPostProcess() {
+    var config = """
+      sources:
+        osm:
+          type: osm
+          url: geofabrik:rhode-island
+          local_path: data/rhode-island.osm.pbf
+      layers:
+      - id: testLayer
+        features:
+        - source: osm
+          geometry: point
+      """;
+    this.planetilerConfig = PlanetilerConfig.from(Arguments.of(Map.of()));
+    assertEquals(null, loadConfig(config).findFeatureLayer("testLayer").postProcess());
```

Comment:
minor: use `assertNull` instead of `assertEquals(null, ...)`

Reply:
Change pushed previously.